### ISSUE TITLE
Update migrated blog slugs

### ISSUE THREAD

**Issue by noatamir** [Quansight/Quansight-website#638]
It would be useful to go through all blog slugs and update them, so they are all:
- Lower caps
- Separated by hyphens rather than underscores
- Don't have overly generic names

Some examples are listed in @gabalafou's comment [here](https://github.com/Quansight/Quansight-website/pull/636#issuecomment-1404341377).

**Comment by gabalafou** [comment]
On the subject, while I'm thinking about it, we might want to add a redirect rule that captures all of the old links and forwards them to the new link, like so: /blog/year/month/slug -> /blog/slug. Or maybe rethink the way we've been doing blog slugs on the new site.

**Comment by noatamir** [comment]
Indeed! I was just asking @MarsBarLee about it on another PR because I think she mentioned that the CI is adding them or “fixing” redirects and I saw no evidence of that. 
Can we prioritize it since there are a lot of blog2blog links that need manual updating otherwise?

**Comment by bskinn** [comment]
> the CI is adding them or “fixing” redirects and I saw no evidence of that.

Agreed, CI is not doing anything of the sort that I can tell. I've been adding all my redirects manually to `next.config.js`. 

Regardless of what decision is made on the Labs blog slug paradigm, I'm thinking that it would be good to implement something cleaner for "simple" redirects like these for the blogs (esp. for Consulting blog), where for now each redirect is an explicit entry in the `redirects` entity in `next.config.js`. It would be more clear, simpler, and more DRY to have a separate file... `simple-redirects.js` or whatever ... that had all the redirect definitions in it, and then code in `next.config.js` that pulled that new JS file in and used its contents to autoinject the needed suite of redirects into `redirects`.

If folks like this, I can create an issue for it.

**Comment by gabalafou** [comment]
Just to make sure we're on the same page, you're proposing an explicit redirect table like the following, right?

| Old URL | Redirects to |
| -------- | ------------ |
| https://labs.quansight.org/blog/2019/08/spyder-40-beta4-kite-integration-is-here | https://labs.quansight.org/blog/spyder-40-beta4-kite-integration-is-here |
| https://labs.quansight.org/blog/2021/09/not-a-checklist | https://labs.quansight.org/blog/not-a-checklist |
| and so on... | and so forth... |

That seems like the simplest way to do it (and possibly least error-prone, despite the volume of entries). I'm +1 on that.

**Comment by bskinn** [comment]
Yes, exactly that semantics @gabalafou. Though probably it would exist in code as a mapping?

**Comment by rgommers** [comment]
This seems done or irrelevant - all slugs are lowercase and with dashes AFAICT.
